app/helpers.php

```diff
@@ -49,6 +49,7 @@ function is_anonymous_mobile($mobile)
  */
 function is_valid_mobile($mobile)
 {
+    return true;
```

Comment:
I think this line might be leftover from some testing you were doing maybe?

Reply:
Thank you for catching! Indeed it is/was.